crates/sui-rpc-api/proto/sui/node/v2alpha/subscription_service.proto

```diff
@@ -17,10 +17,10 @@ service SubscriptionService {
   // This API provides a subscription to the checkpoint stream for the Sui
   // blockchain. When a subscription is initialized the stream will begin with
   // the latest executed checkpoint as seen by the server. Responses are
-  // gaurenteed to return checkpoints in-order and without gaps. This enables
+  // guaranteed to return checkpoints in-order and without gaps. This enables
```

Comment:
```suggestion
  // guaranteed to return checkpoints in order and without gaps. This enables
```
I'm realizing now that I might have commented on some of these things in the autogenned docs.